mdnshosts/Makefile

```diff
@@ -0,0 +1,18 @@
+PREFIX?=/usr/local
+BINDIR=${PREFIX}/bin
+
+PROG=	mdnshosts
+SRCS=	main.c mdnsl.c 
+
+CFLAGS+= -g -Wall
+CFLAGS+= -Wstrict-prototypes -Wmissing-prototypes
+CFLAGS+= -Wmissing-declarations
+CFLAGS+= -Wshadow -Wpointer-arith -Wcast-qual
+CFLAGS+= -Wsign-compare
+CFLAGS+= -I${.CURDIR} -I${.CURDIR}/../mdnsd
```

Comment:
here could add
CFLAGS += -I{.CURDIR}/../mdnsctl/

so then you can get mdnsl.c